lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1177,6 +1177,14 @@ std::optional<std::vector<std::string>> ClangImporter::getClangCC1Arguments(
     llvm::SmallVector<const char *> invocationArgs;
     invocationArgs.reserve(driverArgs.size());
     llvm::for_each(driverArgs, [&](const std::string &Arg) {
+      if (ctx.LangOpts.Target.isOSWindows() &&
+          ctx.LangOpts.CXXStdlib != ctx.LangOpts.PlatformDefaultCXXStdlib &&
```

Comment:
You could also use `ctx.LangOpts.isUsingPlatformDefaultCXXStdlib()` to slightly simplify this.